en/docs/administration/production-deployment-guideline.md

```diff
@@ -0,0 +1,72 @@
+# Production Deployment guideline
+
+The requirements for deploying WSO2 products can vary based on the deployment scenario and pattern. The recommendations in this topic are intended for general production use.
+
+WSO2 APK can be configured through `values.yaml` file.  Please refer to [Customize Configurations](../setup/Customize-Configurations.md) for information on how to use a customized values file for APK deployment. When deploying WSO2 APK in a production environment, we strongly recommend following these guidelines.
+
+
+## Change the hostnames and vhosts
+
+By default, APK uses wso2.com for its hostnames and vhosts for the gateway. You need to change these values to your own domain, which you plan to use for production. The following values.yaml values should be modified:
+
+- wso2.apk.listener.hostname
+- wso2.apk.dp.gateway.listener.hostname
+- wso2.apk.dp.configdeployer.vhosts
```

Comment:
In this case we should say what are the hosts we have in APK and what are their use cases. For example, api.wso2.com is used for the gateway and it is used to invoke the APIs, etc. 

After defining these, we can say if you want to change the gateway host, change wso2.apk.dp.gateway.listener.hostname.

Reply:
Maybe we can come up with a table.

Reply:
Updated